docs/workforce-management-framework/topics/ref-passportauth-securitystrategy.adoc

```diff
@@ -30,19 +30,17 @@ app.get('/testAdminEndpoint', authService.protect('admin'), (req: express.Reques
 });
 ----
 
-NOTE: Demo roles are defined as an array in link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/wfm-user/users.json[user.json].
+NOTE: Demo roles are defined in link:{WFM-RC-CoreURL}{WFM-RC-Branch}/demo/server/src/modules/wfm-user/users.json[user.json].
 
 [id='defining-password-storage-{chapter}']
 == Defining Password Storage
-The link:../../../api/{WFM-RC-Api-Version}{WFM-RC-Api-User-Repository}#getuserbylogin[User Repository Interface] 
-is used to store *PassportAuth* user data.
-To point *PassportAuth* to a datasource, you must implement the User Repository Interface.
+The *PassportAuth* user data is stored in link:../../../api/{WFM-RC-Api-Version}{WFM-RC-Api-User-Repository}#getuserbylogin[User Repository Interface].
+User Repository Interface must be implemented in order to point *PassportAuth* to a datasource,
 
 [id='passportauth-authentication-{chapter}']
 == PassportAuth Authentication
-{WFM-RC-NameShort} uses the Default Strategy for username/password authentication, and authentication happens on 
-the cloud server. The *PassportAuth* authentication service uses a default custom strategy that is based on the 
-local *Passport.js* strategy.The *PassportAuth* local strategy is database agnostic.
+{WFM-RC-NameShort} uses username/password as the default authentication strategy, credentials are authenticated on the cloud server.
```

Comment:
uses "a username and password combination" as the..


authentication strategy "and the" credentials are authenticated on the cloud server.